config/docker/nginx.conf

```diff
@@ -135,6 +135,17 @@ server {
     proxy_set_header X-Forwarded-For $remote_addr;
   }
   
+  location /cards-external-diffusion/ {
+    proxy_set_header Host $http_host;
+    proxy_pass http://cards-external-diffusion:2106/;
+    proxy_set_header X-Forwarded-For $remote_addr;
+  }
+
+  location /cards-reminder/ {
+    proxy_set_header Host $http_host;
+    proxy_pass http://cards-reminder:2107/;
+    proxy_set_header X-Forwarded-For $remote_addr;
+  }
```

Comment:
Add this modification in migration guide : https://opfab.github.io/documentation/archives/SNAPSHOT/docs/single_page_doc.html#_endpoints_configuration

Add a bullet point with a text like  : Add services cards-reminder and cards-external diffusion accessible via nginx for opfab cli